qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -341,328 +345,426 @@ def __init__(self, name, setup_folder, address, reset=False,
 
     # Functions
     def get_all(self, update=True):
```

Comment:
No good reason. I left it in case of compatibility issues, but I agree that it is a silly wrapper/renaming for a standard function.